concent-builder/configure-and-run-tests.yml

```diff
@@ -0,0 +1,55 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - repositories.yml
+    - ../containers/versions.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: jenkins
+      block:
+        - name: Copy concent and virtualmachine for deployment build
+          command: cp                                                                  \
+              --recursive                                                              \
+              "{{ build_dir }}/concent-deployment/containers/build/{{ item.path }}"    \
+              "{{ jenkins_home_dir }}/{{ item.name }}"
+          with_items:
+            - { path: "repositories/concent", name: "concent" }
+            - { path: "virtualenv", name: "virtualenv" }
+
+        - name: Copy local_settings.py file
+          copy:
+            src:        "{{ jenkins_home_dir }}/concent-deployment/concent-builder/jenkins-configuration-files/local_settings.py"
+            dest:       "{{ jenkins_home_dir }}/concent/concent_api/concent_api/settings/local_settings.py"
+            owner:      jenkins
+            group:      jenkins
+            remote_src: yes
+
+    - become:      yes
+      become_user: "{{ shared_user }}"
+      block:
+        - name: Wait for geth synchronization
```

Comment:
Jenkins should not have to run anything as a different user. This would require sudo access.

Reply:
I think that it's not possible to do, maybe after this pull request #223 .

Reply:
Then we have to do #223 first. I has already been reviewed by me and is waiting for corrections.

Reply:
I this done? I don't see this in the code any more.

Reply:
This is not part of code any more.